### PR TITLE
GIN-21-Copyright-section-implementation

### PR DESCRIPTION
**Pull request by OnurRozet**
Copyright section completed

### CODE REVIEW

**Review by asozyurt — CHANGES_REQUESTED** [review]
I should only see the copyright changes here. it is showing tons of changes.

**Comment by OnurRozet** [comment]
Copyright Section fixed. Only changed CopyRight

File: public/index.html [hunk 1]
```diff
@@ -1,862 +1,1042 @@
...
+    <script src="assets/js/backtotop.js"></script>
+    <script src="assets/js/nice-select.js"></script>
+    <script src="assets/js/counterup.js"></script>
+    <script src="assets/js/wow.js"></script>
+    <script src="assets/js/isotope-pkgd.js"></script>
+    <script src="assets/js/imagesloaded-pkgd.js"></script>
+    <script src="assets/js/ajax-form.js"></script>
+    <script src="assets/js/main.js"></script>
+    <script>
+      let date = new Date().getFullYear();
```
Comment (asozyurt) [inline public/index.html]: no need to use let here

File: public/index.html [hunk 2]
```diff
@@ -1,862 +1,1042 @@
...
+      let dom = (document.querySelector(".date").innerHTML = date);
```
Comment (asozyurt) [inline public/index.html]: why did you define dom ? it is not being used anywhere. Check the scenario that; the query selector cannot find the item. What happens then?

**Review by asozyurt — CHANGES_REQUESTED** [review]

**Review by asozyurt — APPROVED** [review]

**Pull request merged**